contrib/IECoreMantra/test/IECoreMantra/RendererTest.py

```diff
@@ -196,12 +196,13 @@ def testShaderParameters( self ):
 		self.assertTrue( world )
 		self.assertEquals( world.typeId(), IECore.Group.staticTypeId() )
 		self.assertTrue( world.state() )
-		self.assertEquals( 
-			world.state()[0].attributes[':surface'], 
-			IECore.StringData( 'testshader p2 1.234 p3 "hello" p1 11 p4 1 2 3 p5 1 0 0 ')
+		self.assertItemsEqual(
```

Comment:
Yes, there was a different order. I guess if they're arguments then they do need to be in a specific order.